ui/components/app/snaps/snap-ui-input/snap-ui-input.tsx

```diff
@@ -54,10 +56,13 @@ export const SnapUIInput: FunctionComponent<
       ref={inputRef}
       onFocus={handleFocus}
       onBlur={handleBlur}
-      className="snap-ui-renderer__input"
+      className={classnames('snap-ui-renderer__input', {
+        'snap-ui-renderer__field': label !== undefined,
+      })}
       id={name}
       value={value}
       onChange={handleChange}
+      {...(label ? { label } : {})}
```

Comment:
```suggestion
      label={label}
```

Any reason not to do this?